src/Datadog.Trace/Configuration/TracerSettings.cs

```diff
@@ -170,9 +170,20 @@ public static CompositeConfigurationSource CreateDefaultConfigurationSource()
 #endif
             };
 
+            string currentDirectory = System.Environment.CurrentDirectory;
+
+#if !NETSTANDARD2_0
+            // on .NET Framework only, use application's root folder
+            // as default path when looking for datadog.json
+            if (System.Web.Hosting.HostingEnvironment.IsHosted)
+            {
+                currentDirectory = System.Web.Hosting.HostingEnvironment.MapPath("~");
+            }
```

Comment:
Would this merit a fallback if it's not found?

Reply:
The file is optional and we already check [below](https://github.com/DataDog/dd-trace-dotnet/pull/463/files/c4e9cd8ae8d6a31a055bf6e21e4d658810218dba#diff-eee563ad9ab37c7a6bf90b77f1d73adbR188) if it exists before adding a `JsonConfigurationSource` to the list.